### DISCUSSION TITLE
Honey gain configuration in multiple proxies

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#146] [Ideas]
Hi all,

Anyone can help me to setup multiple accounts with multiple proxies in the same machine. As I know, Honeygain allowed 10 IP/account, so I cant setup more. 
I have an idea that I will clone the directory of InternetIncome into multiple directory and then I will setup one by one. Anyone has better ideas.

Thanks

**Comment by engageub** [comment]
Hello,
This has been answered in FAQ section regarding multiple accounts. Please find the link below. 

https://github.com/engageub/InternetIncome/wiki/Frequently-Asked-Questions#-how-to-use-multiple-accounts

Thank you

**Comment by ByQwank** [comment]
> Hi all,
> 
> Anyone can help me to setup multiple accounts with multiple proxies in the same machine. As I know, Honeygain allowed 10 IP/account, so I cant setup more. I have an idea that I will clone the directory of InternetIncome into multiple directory and then I will setup one by one. Anyone has better ideas.
> 
> Thanks

Add me via discord, I can help set you up. Dylaaann123
